content/en/Platform Deep Dive/Scans/faq.md

```diff
@@ -0,0 +1,72 @@
+---
+title: "FAQ"
+linkTitle: "FAQ"
+weight: 70
+description: >
+  Frequently asked questions about the DAST scanner.
+---
+
+## How does the crawler work?
+The crawler starts with the target URL and any configured seed paths. It tries to follow and click all the links on a given page. The new pages and new injection points found are added to the list of URLs to scan. The crawler can detect patterns in the URLs and mark some URLs as duplicates. Some of the duplicates will be sampled to be scanned as well.
```

Comment:
Could link 'seed paths' to platform-deep-dive/scans/targets/#configuring-a-target